src/Symfony/Component/Serializer/Encoder/YamlEncode.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Encoder;
+
+use Symfony\Component\Yaml\Dumper;
+
+class YamlEncode implements EncoderInterface
+{
+
+    /**
+     * @var \Symfony\Component\Yaml\Dumper
+     */
+    protected $dumper;
```

Comment:
internal properties should always be private in Symfony, unless we make an explicit choice for the opposite